assets/scss/components/_footer.scss

```diff
@@ -0,0 +1,36 @@
+.footer {
+
+    h3 {
```

Comment:
Be careful about using HTML as a selector.  It isn't inherently bad, but it does make your HTML somewhat rigid.